frontend/src/metabase-types/api/dataset.ts

```diff
@@ -38,7 +38,7 @@ export interface DatasetColumn {
   source: string;
   aggregation_index?: number;
 
-  aggregation_type?: AggregationType;
+  aggregation_type?: "sum" | "avg" | "min" | "max" | "stddev" | "median";
```

Comment:
So these 6 values are a subset of `AggregationType`. How did you verify that we don't need all the values from AggregationType?